src/remote.c

```diff
@@ -202,7 +202,8 @@ int dcc_compile_remote(char **argv,
                        pid_t cpp_pid,
                        int local_cpu_lock_fd,
                        struct dcc_hostdef *host,
-                       int *status)
+                       int *status,
+                       int hasgcov)
```

Comment:
const hasgcov?

I have now remembered that this is C and possibly not C99  (and certainly not C++ so my comments about bool use are probably invalid)